wordpress-mobile/WordPress-Android
# Add indicator line to selected value in overview block

Fixes #9061 
This PR adds a line to the selected type in the Overview blocks. Constraint layout didn't work in this case so I moved the columns to a separate linear layout. I moved the padding back to the columns so be careful with different languages.

To test:
* Go to Stats/DWMY tab
* Notice the selected type in overview block is underlined
* Click on a different type
* The indicator is now on the newly selected type

To test 2: 
* Go to Stats/Insights
* The Latest post summary block doesn't show indicators

To test 3:
* Switch the language to Spanish
* Go to Stats/DWMY tab
* The paddings in the Overview block columns are not broken

**Default font size and display size:** 

![screenshot_1548948283](https://user-images.githubusercontent.com/1079756/52064404-e2062b80-2574-11e9-8938-1aa08fc141eb.png)

**Largest font size and display size**

![screenshot_1548948258](https://user-images.githubusercontent.com/1079756/52064415-e7fc0c80-2574-11e9-8cf5-ae615f76f6de.png)
